cli/src/main/java/com/adaptivescale/rosetta/cli/Cli.java

```diff
@@ -324,6 +323,54 @@ private void init(@CommandLine.Parameters(index = "0", description = "Project na
         if (!projectName.isEmpty()) {
             log.info("In order to start using the newly created project please change your working directory.");
         }
+
+
+        Path driversPath = Path.of(DEFAULT_DRIVERS_YAML);
+
+        DriverHelper.printDrivers(driversPath);
+        System.out.println("Which is the source DB you plan to use (default: skip)?");
+        String sourceDB = new BufferedReader(new InputStreamReader(System.in)).readLine().trim();
+        if (sourceDB.isEmpty()) {
+            sourceDB = "skip";
+        }
+        handleDriverDownload(sourceDB, driversPath, "source");
+
+        DriverHelper.printDrivers(driversPath);
+        System.out.println("Which is the target DB you plan to use (default: skip)?");
+        String targetDB = new BufferedReader(new InputStreamReader(System.in)).readLine().trim();
+        if (targetDB.isEmpty()) {
+            targetDB = "skip";
+        }
+        handleDriverDownload(targetDB, driversPath, "target");
+    }
+
+    private void handleDriverDownload(String dbChoice, Path driversPath, String dbType) {
+        if ("skip".equalsIgnoreCase(dbChoice)) {
+            log.info("Skipped downloading the {} DB driver.", dbType);
+            return;
+        }
+
+        Integer driverId = null;
+
+        try {
+            driverId = Integer.parseInt(dbChoice);
+        } catch (NumberFormatException e) {
+            System.out.println("Invalid index. Please provide a valid index.");
```

Comment:
/nit
Presuming this for when user is provided numbered option to select from:
Invalid choice. Please select a valid option.